### DISCUSSION TITLE
Hue spot 5047330P6 'not supported'?

### DISCUSSION THREAD

**Discussion by bplatypus** [Koenkk/zigbee2mqtt#18712] [Devices]
Hi,
I have 2 Hue white ambiance Buckram spot, 1 double-spot and 1 triple-spot. The double spot is working OK with z2m but the spots from the triple spot are detected but marked as 'not supported'.

The weird thing is that the spot was not detected by the hue app... I had to enter the serial number of each spot to make them work... then I updated them to latest firmware (1.104.2) and reset them to re-try detection with z2m... but it is still 'not supported' and same behaviour in hue app.. nothing is detected... I have to enter the serial numbers again to make them work.

Any way I can make those work in z2m?

It is strange that the double-spot (which has the same spots, with the same firmware (1.104.2) but different zigbee model (!) ... 929003047901_01 & 929003047901_02) are working fine.

Thanks for your help!

**Comment by Koenkk** [comment]
Can you provide the data/database.db entry of this device?

**Comment by bplatypus** [comment]
**'Not Supported' spot (wrong SW detected? 1.65.11)**
{"id":17,"type":"Router","ieeeAddr":"0x00178801096fa1c4","nwkAddr":8423,"manufId":4107,"manufName":"Philips","powerSource":"Mains (single phase)","modelId":"5047330P6","epList":[11,242],"endpoints":{"11":{"profId":260,"epId":11,"devId":268,"inClusterList":[0,3,4,5,6,8,4096,64515,768,64516],"outClusterList":[25],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":2,"stackVersion":1,"hwVersion":3,"dateCode":"20200327","swBuildId":"1.65.11_hB798F2B","zclVersion":2,"interviewCompleted":true,"meta":{},"lastSeen":1692358608422,"defaultSendRequestWhen":"immediate"}

**Working spot (with right SW detected 1.104.2)**
{"id":22,"type":"Router","ieeeAddr":"0x001788010d4b21c6","nwkAddr":3972,"manufId":4107,"manufName":"Philips","powerSource":"Mains (single phase)","modelId":"5062131P7","epList":[11,242],"endpoints":{"11":{"profId":260,"epId":11,"devId":269,"inClusterList":[0,3,4,5,6,8,4096,64514,768,64513],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"5062131P7","manufacturerName":"Philips","powerSource":1,"zclVersion":2,"appVersion":2,"stackVersion":1,"hwVersion":3,"dateCode":"20230403","swBuildId":"1.104.2"}},"lightingColorCtrl":{"attributes":{"colorCapabilities":31,"colorTempPhysicalMin":153,"colorTempPhysicalMax":500}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":2,"stackVersion":1,"hwVersion":3,"dateCode":"20230403","swBuildId":"1.104.2","zclVersion":2,"interviewCompleted":true,"meta":{"configured":2},"lastSeen":1692860375323,"defaultSendRequestWhen":"immediate"}

And here you have the 'not supported' spot in the Hue app:
https://i.imgur.com/6jsYnnu.jpg

**Reply by Koenkk** [reply]
Added, device will be supported in the 1 September release.

**Reply by bplatypus** [reply]
Awesome! Thanks.
